docs/administration-guide/databases/oracle.md

```diff
@@ -1,67 +1,52 @@
-## Working with Oracle in Metabase
+# Working with Oracle in Metabase
 
-Starting in v0.20.0, Metabase provides a driver for connecting to Oracle databases. Under the hood, Metabase uses Oracle's JDBC driver; due to licensing restrictions, we can't
-include it as part of Metabase. Luckily, downloading it yourself and making it available to Metabase is straightforward and only takes a few minutes.
+Under the hood, Metabase uses Oracle's JDBC driver, but due to licensing restrictions, we can't include Oracle's driver as part of Metabase, so you'll need to download the driver and make it available to Metabase.
```

Comment:
I actually don't think that's true anymore after Oracle suddenly became less :hankey:heads and started releasing under a different license last year, but I'm not a lawyer and Oracle is known for being in the top 3 of worst licensing company on Earth.